packages/host/app/components/operator-mode/code-mode.gts

```diff
@@ -530,6 +534,14 @@ export default class CodeMode extends Component<Signature> {
     return undefined;
   }
 
+  private get isSaving() {
+    return (
+      (isReady(this.openFile.current) && this.openFile.current.isWriting) ||
+      this.saveFileSerializedCard.isRunning ||
+      this.saveCard.isRunning
```

Comment:
this is all the async related to actually persisting source files and card instances